helm/alfresco-content-services/values.yaml

```diff
@@ -66,7 +66,11 @@ repository:
       memory: "3000Mi"
     limits:
       memory: "3000Mi"
-  # -- The repository readiness probe is used to check startup only as a failure
+  # -- The startup probe to cover the worse case startup time for slow start containers
+  startupProbe:
+    periodSeconds: 30
+    failureThreshold: 10
+  # The repository readiness probe is used to check startup only as a failure
```

Comment:
see my comment in conversation.
tl;dr: I think this should not go to the main container